src/lib/helpers.ts

```diff
@@ -0,0 +1,29 @@
+import { DataSetColumnType } from './dataset';
+
+function issStringBoolean(string: string): boolean {
+  return (
+    string === 'true' ||
```

Comment:
just lowercase the string and then compare to `true` / `false`. It will catch strings like `trUe` but is that really a problem?